city_scrapers/spiders/cook_north_shore_mosquito.py

```diff
@@ -0,0 +1,89 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookNorthShoreMosquitoSpider(CityScrapersSpider):
+    name = "cook_north_shore_mosquito"
+    agency = "Cook County North Shore Mosquito Abatement District"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.nsmad.com"]
+    start_urls = ["https://www.nsmad.com/news-events/board-meetings/"]
+    TAG_RE = re.compile(r'<[^>]+>')
+    location = {
+        "address": "117 Northfield Road, Northfield, IL 60093",
+        "name": "NSMAD Office",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._parse_location(response)
+        calendar = response.xpath('//*[@id="post-68"]/div/ul/li/ul')
+        for index, item in enumerate(calendar.extract()[0].split('<li>')[1:]):
+            print(index)
```

Comment:
Is this ready for review again @mattpair? Other than removing this print statement it looks good to go